piq/perceptual.py

```diff
@@ -104,7 +104,7 @@ class ContentLoss(_Loss):
         https://arxiv.org/abs/1801.03924
     """
 
-    def __init__(self, feature_extractor: Union[str, Callable] = "vgg16", layers: Tuple[str] = ("relu3_3", ),
+    def __init__(self, feature_extractor: Union[str, torch.nn.Module] = "vgg16", layers: Iterable[str] = ("relu3_3", ),
```

Comment:
I would leave `Callable` instead of `torch.nn.Module` since it is possible that a user would want to use some function which not only includes forward through a `torch.nn.Module` extractor but also some pre- or post-processing

Reply:
Just a function won't work in current implementation, because we have calls like:
`self.model.parameters()`, `self.model.to(prediction)`, `self.model._modules`. Easiest way to add custom preprocessing is inherit class and modify `forward` step.

Reply:
Ok, makes sense. Let's then leave it this way.